go.mod

```diff
@@ -6,7 +6,10 @@ go 1.21
 
 toolchain go1.21.3
 
-require go.mondoo.com/cnquery/v9 v9.10.0
+require (
+	github.com/aws/aws-sdk-go-v2/service/s3 v1.47.3
```

Comment:
I think we should keep the dep in the second group, makes it easier to understand